packages/react-components/src/components/SendBox.tsx

```diff
@@ -217,12 +242,36 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
   };
 
   /* @conditional-compile-remove(file-sharing) */
-  const renderFileUploadErrorMessage: JSX.Element = useMemo(() => {
+  const uploadInProgressErrorBar = useMemo(() => {
```

Comment:
I'd recommend limiting this to a boolean `hasIncompleteUploads`. You can also add a helper file-level function for this to minimize the conditional compilation (that function will always return `false` on stable builds).

Then, you can use that boolean value to decide both whether to show the `SendBoxErrorBar` (hopefully encpasulated in its own component to handle all the timeouts) and whether to skip sending the file.